climakitae/selectors.py

```diff
@@ -651,7 +651,7 @@ def _get_simulation_options(cat, activity_id, table_id, grid_label, experiment_i
 
     # Get all unique simulation options from catalog selection
     try:
-        simulation_options = cat_subset.unique()["source_id"]
+        simulation_options = cat_subset.unique()["source_id"]["values"]
```

Comment:
You could use the `DataFrame` here to get unique vals specific to each column, e.g.
``` python
cat_subset.df['source_id'].unique()
```
This has the advantage of the syntax working on both images, plus it's more efficient calling `.unique()` per column instead of the entire catalog for every column.

Reply:
great idea, thanks!